app/server/src/main/scala/org/bitcoins/server/Main.scala

```diff
@@ -322,6 +323,32 @@ object Main extends App with HttpLogger {
       start
     }
   }
+
+  def createWalletWithBitcoindCallbacks(
```

Comment:
nit: Maybe move this out into a separate file and possibly add a test case for it? Maybe this could go into your `BitcoindRpcBackendUtil`